Microsoft.Toolkit.Uwp.UI.Animations/Implicit.cs

```diff
@@ -0,0 +1,249 @@
+        /// <returns><see cref="AnimationCollection"/></returns>
+        public static AnimationCollection GetHideAnimations(DependencyObject obj)
+        {
+            var collection = (AnimationCollection)obj.GetValue(HideAnimationsProperty);
+
+            if (collection == null)
+            {
+                collection = new AnimationCollection();
+                obj.SetValue(HideAnimationsProperty, collection);
+            }
+
+            return collection;
+        }
+
+        /// <summary>
+        /// Sets the value of the Implicit.HideAnimations XAML attached property.
+        /// </summary>
+        /// <param name="obj">The <see cref="FrameworkElement"/> to set the value</param>
+        /// <param name="value">The <see cref="AnimationCollection"/> to set</param>
+        public static void SetHideAnimations(DependencyObject obj, AnimationCollection value)
+        {
+            obj.SetValue(HideAnimationsProperty, value);
+        }
+
+        /// <summary>
+        /// Gets the value of the Implicit.Animations XAML attached property.
+        /// </summary>
+        /// <param name="obj">The <see cref="FrameworkElement"/> to get the value from</param>
+        /// <returns><see cref="AnimationCollection"/></returns>
+        public static AnimationCollection GetAnimations(DependencyObject obj)
+        {
+            var collection = (AnimationCollection)obj.GetValue(AnimationsProperty);
+
+            if (collection == null)
+            {
+                collection = new AnimationCollection();
+                obj.SetValue(AnimationsProperty, collection);
+            }
+
+            return collection;
+        }
+
+        /// <summary>
+        /// Sets the value of the Implicit.Animations XAML attached property.
+        /// </summary>
+        /// <param name="obj">The <see cref="FrameworkElement"/> to set the value</param>
+        /// <param name="value">The <see cref="AnimationCollection"/> to set</param>
+        public static void SetAnimations(DependencyObject obj, AnimationCollection value)
+        {
+            obj.SetValue(AnimationsProperty, value);
+        }
+
+        private static void ShowAnimationsChanged(DependencyObject d, DependencyPropertyChangedEventArgs e)
+        {
+            if (!AnimationBase.IsCreatorsUpdateOrAbove)
+            {
+                return;
+            }
+
+            if (e.OldValue is AnimationCollection oldCollection)
+            {
+                oldCollection.AnimationCollectionChanged -= ShowCollectionChanged;
+            }
+
+            if (e.NewValue is AnimationCollection animationCollection && d is UIElement element)
+            {
+                animationCollection.Element = element;
+                animationCollection.AnimationCollectionChanged -= ShowCollectionChanged;
+                animationCollection.AnimationCollectionChanged += ShowCollectionChanged;
```

Comment:
Why - then + ? This seems like a "just in case" kind of thing.
Also these event handlers should be weak.

Reply:
Yup, the same AnimationCollection can be added on multiple elements. But there is no need to do this in those cases, so I'll remove these.